docs/content/languages/java.md

```diff
@@ -228,6 +228,42 @@ For function of executor type "newdeploy" you can also override the resource val
 $ fission fn create --name javatest --pkg  java-src-pkg-zip-dqo5 --env java --entrypoint io.fission.HelloWorld --executortype newdeploy --minscale 1 --maxscale 1  --mincpu 100 --maxcpu 500 --minmemory 128 --maxmemory 512
 ```
 
+## JVM Parameters
+
+You can provide options to JVM such as heap size or additional parameters for tuning the JVM to your needs. The Fission JVM environment supports the JVM options.
+
+You can create the JVM environment spec and then add environment variable named `JVM_OPTS`. The value of environment variable is used as options to JVM when function starts.
+
+```bash
+$ 
+
+```
+
+```yaml
```

Comment:
We need some extra information/instruction to indicate the following YAML is used for fission spec.